test/unit/std/charconv_int_test.cpp

```diff
@@ -231,14 +233,19 @@ TYPED_TEST(integral_from_char_test, hexadicimal_number)
 
 TYPED_TEST(integral_from_char_test, to_chars)
 {
-    TypeParam val{120};
-    std::array<char, 10> buffer{};
+    uint8_t max_num_digits = static_cast<uint8_t>(std::log10(std::numeric_limits<TypeParam>::max())) + 1;
+    std::array<char, 20> buffer{};
 
-    auto res = std::to_chars(buffer.data(), buffer.data() + buffer.size(), val);
+    for (uint8_t num_digits = 1; num_digits <= max_num_digits; ++num_digits)
+    {
+        TypeParam val = seqan3::pow(10, num_digits - 1);
 
-    EXPECT_EQ(res.ptr, &buffer[3]);
-    EXPECT_EQ(res.ec, std::errc{});
-    EXPECT_EQ((std::string_view{buffer.data(), 3}), std::string_view{"120"});
+        auto res = std::to_chars(buffer.data(), buffer.data() + buffer.size(), val);
+
+        EXPECT_EQ(res.ptr, &buffer[num_digits]);
+        EXPECT_EQ(res.ec, std::errc{});
+        EXPECT_EQ((std::string_view{buffer.data(), num_digits}), std::string_view{std::to_string(val)});
+    }
 }
 
 TYPED_TEST(integral_from_char_test, to_chars_error)
```

Comment:
Can you add the regression test?

```c++
TEST(to_chars_test, large_number)
{
    uint64_t val{123456789};
    std::array<char, 100> buffer{};

    auto res = std::to_chars(buffer.data(), buffer.data() + buffer.size(), val);

    EXPECT_EQ(res.ptr, &buffer[9]);
    EXPECT_EQ(res.ec, std::errc{});
    EXPECT_EQ((std::string_view{buffer.data(), 9}), std::string_view{"123456789"});
}
```
? Thank you :)

Reply:
Well, it's kinda enclosed in the generic test, because it breaks for every value in `[100000000, 10000000000000000)`, so `100000000` is also a regression test.

Reply:
Yeah, but a regression test is a test that is exactly like the bug report stated. We want to make sure that that particular instance didn't work, but now works. With this, we can always guarantee that that particular use case is working with our code.

But of course, we should also write generic tests that will cover those regressions.